### PR TITLE
Feature/il 4135 wusdm arbitrum

### PR DESCRIPTION
**Pull request by pete-ipor**

### CODE REVIEW

File: test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+    address userOne;
+
+    function setUp() public {
+        _init();
+        userOne = _getUserAddress(22);
+        _setupUser(userOne, 100_000 * 1e18);
+    }
+
+    function testtt() public {
```
Comment (amydlarz) [inline test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol:18]: ???

File: test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        assertTrue(true);
+    }
+
+    function testShouldExchangeRateBe1WhenNoProvideStEth() external {
+        //given
+
+        //when
+        uint exchangeRate = IAmmPoolsLensWusdm(IporProtocolRouterProxy).getIpWusdmExchangeRate();
```
Comment (amydlarz) [inline test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol:26]: IAmmPoolsLensWusdm(IporProtocolRouterProxy) is given

File: test/amm-wusdm/arbitrum/WusdmForkAmmGovernanceServiceArbitrumTest.t.sol [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.20;
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import "./WusdmTestForkCommonArbitrum.sol";
+import "../../../contracts/interfaces/IAmmGovernanceService.sol";
+
+contract WusdmForkAmmGovernanceServiceArbitrumTest is WusdmTestForkCommonArbitrum {
+    function setUp() public {
+        _init();
```
Comment (amydlarz) [inline test/amm-wusdm/arbitrum/WusdmForkAmmGovernanceServiceArbitrumTest.t.sol:10]: not necessary if you have _init in each test

**Review by amydlarz — APPROVED** [review]

File: test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol (hunk 2 above)
Comment (pete-ipor) [reply test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol:26]: this test check init value

**Pull request merged**